build/critical.go

```diff
@@ -12,11 +12,11 @@ import (
 // the running goroutine is printed to help determine the error.
 func Critical(v ...interface{}) {
 	s := "Critical error: " + fmt.Sprintln(v...) + "Please submit a bug report here: https://github.com/NebulousLabs/Sia/issues\n"
+	debug.PrintStack()
```

Comment:
I think we should test that the UI is capturing this output correctly, at the moment I don't actually think that it is.